api/proto/core/pipeline/definition/definition.proto

```diff
@@ -91,7 +92,6 @@ message PipelineDefinitionUpdateRequest {
   string executor = 3;
   string pipelineSourceId = 4;
   string category = 5;
-  PipelineDefinitionExtra extra = 6;
   google.protobuf.Timestamp startedAt = 7;
```

Comment:
adjust order

Reply:
After this PR merge, I will propose to change the PR, because the detection time is too long

Reply:
https://github.com/erda-project/erda/pull/3712/files